### DISCUSSION TITLE
Page deployed not getting 404 on assets

### DISCUSSION THREAD

**Discussion by rhuanbarreto** [community/community#42641] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I deployed my static react app using github actions:

https://github.com/rhuanbarreto/rhuanbarreto/actions/runs/3778823773

The tar artifact includes the assets, but the deployed page doesn't.

https://rhuanbarreto.github.io/rhuanbarreto/

**Comment by yoannchaudet** [comment]
Your files are not published at the root of a website:

![image](https://user-images.githubusercontent.com/14911070/209723120-9b9eb2c7-0197-46c2-8a86-946d7cd1d8c8.png)

Either fix all your relative links or rename your repo to `rhuanbarreto.github.io` so it becomes your user site and content is directly accessible at the root of https://rhuanbarreto.github.io.

**Reply by rhuanbarreto** [reply]
Aha! So Vite needs to be configured to use the base url using [this config](https://v2.vitejs.dev/config/#base). Thanks!
Maybe it could be a good idea to add something on github docs to guide people through this.
